src/lib/leagues/eliteTasks.ts

```diff
@@ -0,0 +1,642 @@
+		}
+	},
+	{
+		id: 3055,
+		name: 'Defeat the Emerged Inferno',
+		has: async ({ minigames }) => {
+			return minigames.emerged_inferno >= 1;
+		}
+	},
+	{
+		id: 3056,
+		name: 'Sacrifice 5b GP',
+		has: async ({ sacrificedBank }) => {
+			return sacrificedBank.amount('Coins') >= 5_000_000_000;
+		}
+	},
+	{
+		id: 3057,
+		name: 'Sacrifice 250 different items',
+		has: async ({ sacrificedBank }) => {
+			return sacrificedBank.length >= 250;
+		}
+	},
+	{
+		id: 3058,
+		name: 'Reach 60% CL completion',
+		has: async ({ clPercent }) => {
+			return clPercent >= 60;
+		}
+	},
+	{
+		id: 3059,
+		name: 'Construct 50,000 objects',
+		has: async ({ conStats }) => {
+			return sumArr(conStats.items().map(i => i[1])) >= 50_000;
+		}
+	},
+	{
+		id: 3060,
+		name: 'Chop 50,000 of any logs',
+		has: async ({ woodcuttingStats }) => {
+			return sumArr(woodcuttingStats.items().map(i => i[1])) >= 50_000;
+		}
+	},
+	{
+		id: 3061,
+		name: 'Alch 5000 of any item',
+		has: async ({ alchingStats }) => {
+			return sumArr(alchingStats.items().map(i => i[1])) >= 5000;
+		}
+	},
+	{
+		id: 3062,
+		name: 'Clean 15,000 herbs',
+		has: async ({ herbloreStats }) => {
+			return sumArr(herbloreStats.herbs.items().map(i => i[1])) >= 15_000;
+		}
+	},
+	{
+		id: 3063,
+		name: 'Mix 15,000 unf potions',
+		has: async ({ herbloreStats }) => {
+			return sumArr(herbloreStats.unfPots.items().map(i => i[1])) >= 15_000;
+		}
+	},
+	{
+		id: 3064,
+		name: 'Mix 15,000 potions',
+		has: async ({ herbloreStats }) => {
+			return sumArr(herbloreStats.pots.items().map(i => i[1])) >= 15_000;
+		}
+	},
+	{
+		id: 3065,
+		name: 'Mine 25,000 ores',
+		has: async ({ miningStats }) => {
+			return sumArr(miningStats.items().map(i => i[1])) >= 25_000;
+		}
+	},
+	{
+		id: 3066,
+		name: 'Burn 8000 logs',
+		has: async ({ firemakingStats }) => {
+			return sumArr(firemakingStats.items().map(i => i[1])) >= 8000;
+		}
+	},
+	{
+		id: 3067,
+		name: 'Smith items from 50,000 Adamantite bars',
+		has: async ({ smithingSuppliesUsed }) => {
+			return smithingSuppliesUsed.amount('Adamantite bar') >= 50_000;
+		}
+	},
+	{
+		id: 3068,
+		name: 'Smith items from 30,000 Runite bars',
+		has: async ({ smithingSuppliesUsed }) => {
+			return smithingSuppliesUsed.amount('Runite bar') >= 30_000;
+		}
+	},
+	{
+		id: 3069,
+		name: 'Cast 10,000 spells',
+		has: async ({ spellCastingStats }) => {
+			return sumArr(spellCastingStats.map(i => i.qty)) >= 10_000;
+		}
+	},
+	{
+		id: 3070,
+		name: 'Finish the graceful CL',
+		has: async ({ cl }) => {
+			return gracefulCL.every(i => cl.has(i));
+		}
+	},
+	{
+		id: 3071,
+		name: 'Finish the Queen Black Dragon CL',
+		has: async ({ cl }) => {
+			return queenBlackDragonCL.every(gs => cl.has(gs));
+		}
+	},
+	{
+		id: 3072,
+		name: 'Finish the Vasa Magus CL',
+		has: async ({ cl }) => {
+			return vasaMagusCL.every(gs => cl.has(gs));
+		}
+	},
+	{
+		id: 3073,
+		name: 'Finish the Abyssal Dragon CL',
```

Comment:
Probably change this text to 'Malygos' I don't think most people know that AbyssalDragon = maly.